tests/test_data_remover.py

```diff
@@ -51,3 +55,41 @@ def test_delete_tempfile_ok(fs: FakeFilesystem, caplog: LogCaptureFixture):
             logging.WARNING,
             "File deletion may have failed. Usage of space may increase.",
         ) not in caplog.record_tuples
+
+
+def test_delete_all_ok(capfd: LogCaptureFixture):
+    """Delete all files. - ok"""
+
+    # Create mocker
+    with Mocker() as mock:
+        # Create mocked request - real request not executed
+        mock.delete(
+            DDSEndpoint.REMOVE_PROJ_CONT,
+            status_code=200,
+            json={"message": "All files removed.", "removed": True},
+        )
+
+        with data_remover.DataRemover(authenticate=False, project="project_1") as dr:
+            dr.token = {"Authorization": "Bearer FAKE_TOKEN"}  # required
+            dr.remove_all()
+
+            out, _ = capfd.readouterr()
+            assert "All files removed." in out
```

Comment:
maybe not strictly necessary, but probably good to ensure there's no error too

```suggestion
            out, err = capfd.readouterr()
            assert "All files removed." in out
            assert not err
```

Reply:
it will probably also need
`with pytest.raises(APIError) as err:`

Reply:
Added the check for `not err`

It is not possible to add the `pytest.raises` raises here because that is a check for the exception actually happening, which fails if there is none